pom.xml

```diff
@@ -238,6 +238,8 @@
         <xmltool.version>3.3</xmltool.version>
         <xml-matchers.version>0.10</xml-matchers.version>
         <zookeeper.version>3.4.14</zookeeper.version>
+        <cloudera-manager.version>6.3.0</cloudera-manager.version>
```

Comment:
These should be alphabetical like the rest of the list.

Reply:
Done.